test/views/components/LateSubmissionPenaltySummaryCardSpec.scala

```diff
@@ -740,12 +740,12 @@ class LateSubmissionPenaltySummaryCardSpec extends SpecBase with ViewBehaviours
 
       "set the correct hidden span for a lurking point with no return submitted" in {
         doc.select("a").get(0).ownText() shouldBe "Check if you can appeal"
-        doc.select("span").text() shouldBe "for penalty on late VAT return due 30 October 2069"
+        doc.select("span").get(1).text() shouldBe "for penalty on late VAT return due 30 October 2069"
       }
 
       "set the correct hidden span for a lurking point with a return submitted" in {
         docSubmitted.select("a").get(0).ownText() shouldBe "Appeal this penalty"
-        docSubmitted.select("span").text() shouldBe "for late VAT return due on 30 October 2069"
+        docSubmitted.select("span").get(1).text() shouldBe "for late VAT return due on 30 October 2069"
```

Comment:
This check is a vulnerable to fail or prove the wrong thing:
```suggestion
        docSubmitted.select("a span").get(0).text() shouldBe "for late VAT return due on 30 October 2069"
```